third_party/arcs/examples/VisionPipeline.arcs

```diff
@@ -21,15 +21,14 @@
 // The "source" and "output" particles exist only because "claims" can
 // appear just on outputs and "checks" can appear only on inputs.
 
-particle VideoDataSource
```

Comment:
Here I just changed the name from VideoDataSource to DataSource because this actually is a source of a "model" as well. 

Also, the only reason this particle exists is because arcs has a quirk where claims can only go on outputs, and I need to attach tags to these handles.